### PR TITLE
My frontend functionality is slowing coming together.

### PR DESCRIPTION
**Pull request by hannahmanfredi**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -18,6 +18,9 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.get('/booking', async (req, res) => {
   console.log('inside /booking');
   let campId = parseInt(req.query.campId);
+  if (!campId) {
+    campId = 0;
+  }
   let name = '';
   let init = {};
   await axios.get('http://localhost:3003/overview', { params: { campId: campId } })
```
Comment (kimberlyjk) [inline server/server.js:26]: hey i recognize this 👯

File: client/src/Booking.jsx [hunk 2]
```diff
@@ -42,14 +42,13 @@ class Booking extends React.Component {
...
-    this.book();
-    this.bookingTotal(); //testing
+    this.bookingTotal();
   }
 
   init() {
     axios.get('http://localhost:3002/booking')
     .then(({data}) => {
-      console.log(data)
+      console.log('data: ', data)
```
Comment (kimberlyjk) [inline client/src/Booking.jsx:51]: i see this call being successful on my server side, but none of the date renders on the webpage when i run you front end. 
Screen Shot 2021-05-29 at 10 14 25 AM

File: client/src/Booking.jsx [hunk 3]
```diff
@@ -151,12 +151,13 @@ class Booking extends React.Component {
...
             <div className="row">
               <div className="col-xs-6 check-in-btn">
-                <CheckIn month={this.state.current_month} campId={this.state.campId} click={this.handleClick} submit={this.handleSubmit} />
                 <CheckIn
                   month={this.state.current_month}
                   campId={this.state.campId}
+                  inventory={this.state.inventory}
                   onClick={this.handleClick}
                   submit={this.handleSubmit}
+                  book={this.book}
```
Comment (kimberlyjk) [inline client/src/Booking.jsx:160]: this section is showing up on my front end., which is awesome! nice work getting this calendar to work, I know it was pretty rough math wise for a while ;)

File: client/src/components/CheckInCal.jsx [hunk 4]
```diff
@@ -21,7 +30,69 @@ class CheckInCal extends React.Component {
...
+    preLoad = preLoad.concat(days);
+    const days_needed = (42 - preLoad.length);
+    let next_days = [];
+    next_days = Array.from({length: days_needed}, (_, i) => i + 1);
+    preLoad = preLoad.concat(next_days);
+    console.log(preLoad);
+    return preLoad;
+  }
+
+  getSundays(y) {
```
Comment (kimberlyjk) [inline client/src/components/CheckInCal.jsx:95]: I Alluded to this in a prev comment, but major kudos. you should be v. v. v. proud! I'm super impressed.

**Review by kimberlyjk — APPROVED** [review]
This looks awesome. I'm not sure how far your component should be working. I can see a calendar when i click checkin; however, I can only click the first week, and when i do it looks like ths... 
Screen Shot 2021-05-29 at 10 39 25 AM

This might be the point where you aimed to be at, but I just wanted to let you know this is what I got! 

good work overall! just some minor ocding tweaks and css fun I think you'll be right there.

File: server/server.js (hunk 1 above)
Comment (hannahmanfredi) [reply server/server.js:26]: Yep all my calls to your API work like a charm!

File: client/src/components/CheckInCal.jsx (hunk 4 above)
Comment (hannahmanfredi) [reply client/src/components/CheckInCal.jsx:95]: Ty!!! It was gnarly but ultimately like a semi-intense toy prob.

**Pull request merged**